lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.fg, bold = true }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
```

Comment:
IMO these "identifiers" should either:
1. All be subtle and grey (I prefer this)
2. Or have their respective colors.

@5-pebbles Opinion?

Reply:
I would personally prefer colors as the default.

My repos are usually an unstaged mess, and the colors help prevent the spread of chaos, but that's more of a "me problem".

I think either way works; the people who don't like whatever we pick will just change it in their own configs. Though, it would be worth adding the unchosen variation to the list of common configurations that I plan on writing someday.

**TLDR:** I don't know which is better...

Reply:
which gray do you think should they be?

Reply:
I think most people expect colors.  So go with that.